Meadowlark-js/tests/e2e/automation-setup/e2e.env

```diff
@@ -9,6 +8,11 @@ AUTHORIZATION_STORE_PLUGIN=@edfi/meadowlark-mongodb-backend
 FASTIFY_RATE_LIMIT=true
 FASTIFY_PORT=3000
 MEADOWLARK_STAGE=local
-TOKEN_URL: http://localhost:3000/local/api/oauth/token
 LOG_LEVEL=INFO
 IS_LOCAL=true
+
+OAUTH_SERVER_ENDPOINT_FOR_OWN_TOKEN_REQUEST=http://localhost:3000/local/oauth/token
+OAUTH_SERVER_ENDPOINT_FOR_TOKEN_VERIFICATION=http://localhost:3000/local/oauth/verify
+
+OWN_OAUTH_CLIENT_ID_FOR_CLIENT_AUTH=meadowlark_verify-only_key_1
+OWN_OAUTH_CLIENT_SECRET_FOR_CLIENT_AUTH=meadowlark_verify-only_secret_1
```

Comment:
📢 please note the changes above and make _all_ of the same changes in your fastify `.env` files. @stevenarnold-ed-fi @bradbanister @andonyns